src/environments/environment.ts

```diff
@@ -1,13 +1,14 @@
 // This file can be replaced during build by using the `fileReplacements` array.
 // `ng build --prod` replaces `environment.ts` with `environment.prod.ts`.
 // The list of file replacements can be found in `angular.json`.
+// require('dotenv').config()
 
-require('dotenv').config()
+// const IAM_KEY = process.env.key
 
-const IAM_PUSHER_KEY = process.env.pusher_key
 export const environment = {
+  production: false,
   pusher: {
-    key: IAM_PUSHER_KEY,
+    key: 'IAM_KEY',
```

Comment:
Again, why is this a string? You should be using the `IAM_KEY` value here. Along with the dotenv and process.env.